### PR TITLE
[f-3] chore: add image wrapper styles and update breakout button

### PR DESCRIPTION
**Pull request by frederickfogerty**
This PR begins the work on the styling of the sidebar app to bring it more in line with the intended designs.

image

### Stacked PR Chain: f-3
| PR | Title | Status |  Merges Into  |
|:--:|:------|:-------|:-------------:|
|#93|feat: store current custom attribute value and display img|**Approved**|-|
|#94|chore: create story (inc. empty state) for SidebarApp|**Approved**|#93|
|#95|👉 chore: add image wrapper styles and update breakout button|Pending|#94|

### CODE REVIEW

File: frontend/src/components/SidebarApp.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Imgix from "react-imgix";
 import "../styles/App.css";
 import { IImgixCustomAttributeValue } from "../types/imgixSF";
 import { AddImageIcon } from "./buttons/AddImageIcon";
+import styles from "./SidebarApp.module.css";
```
Comment (luqven) [inline frontend/src/components/SidebarApp.tsx:6]: Assuming we're using `module.css` to component-scope the styles 👏🏼

File: frontend/src/components/SidebarApp.module.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.imageWrapper {
+  background-color: #e3e7eb;
+  height: 160px;
+}
+
+.hr {
```
Comment (luqven) [inline frontend/src/components/SidebarApp.module.css:6]: nit: `hr` doesn't tell us a whole lot about what this is doing. Higher? Height-reset?

**Review by luqven — APPROVED** [review]

File: frontend/src/components/SidebarApp.module.css (hunk 2 above)
Comment (frederickfogerty) [reply frontend/src/components/SidebarApp.module.css:6]: This is in reference to it being a replacement for the `<hr>` tag. I'll update the name to make it clearer

**Comment by frederickfogerty** [comment]
Confirming that CSS modules work on SF 👍 

image

**Pull request merged**